playbooks/appsemblerPlaybooks/update_theme.yml

```diff
@@ -96,6 +94,17 @@
         - install:code
         - update_lms_theme
 
+    # TODO: Probably should figure out why it was broken in the first place? But it's not really a big deal.
+    - name: Fix the theme file owner after git checkout
+      file:
+        dest: "{{ item }}"
+        owner: "{{ edxapp_user }}"
+        group: "{{ edxapp_user }}"
+        recurse: yes
```

Comment:
@OmarIthawi I believe you will need to set `state: directory` for `recurse` to have any effect. 
http://docs.ansible.com/ansible/latest/file_module.html

Reply:
Good catch! Thanks @bryanlandia!